rotkehlchen/chain/ethereum/modules/sky/decoder.py

```diff
@@ -122,155 +115,6 @@ def _decode_migrate_mkr(self, context: DecoderContext) -> DecodingOutput:
         maybe_reshuffle_events([mkr_event, sky_event], context.decoded_events)
         return DEFAULT_DECODING_OUTPUT
 
-    def _decode_susds_events(self, context: DecoderContext) -> DecodingOutput:
-        if not (
-            (is_deposit := context.tx_log.topics[0] == DEPOSIT_USDS) or
```

Comment:
can you add in the docstring of this class what the Sky decoder handles exactly? Likewise for the Spark decoder since we will have this issue again with us not knowing what each one does